pkgs/applications/science/electronics/simulide/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, fetchbzr
+, mkDerivation
+, qmake
+, qtserialport
+, qtmultimedia
+, qttools
+, qtscript
+}:
+
+let
+  version = "0.4.15";
+  release = "SR10";
+  branch = "simulide_0.4.14"; # the branch name does not mach the version for some reason
+  rev = "291";
+  sha256 = "sha256-BBoZr/S2pif0Jft5wrem8y00dXl08jq3kFiIUtOr3LM=";
+in
+mkDerivation {
+  pname = "simulide";
+  version = "${version}-${release}";
+
+  src = fetchbzr {
+    url = "https://code.launchpad.net/~arcachofo/simulide/${branch}";
+    inherit rev sha256;
+  };
+
+  postPatch = ''
```

Comment:
```suggestion
  postPatch = ''
    # GCC 13 needs this header explicitly included
    sed -i src/gpsim/value.h -e '1i #include <cstdint>'
    sed -i src/gpsim/modules/watchdog.h -e '1i #include <cstdint>'
```

Reply:
I tried doing `env.CXXFLAGS = "-include cstdint";` but it didn't seem to work. So I came up with this instead, inspired by other packages that had to patch GCC 13

Reply:
Okay, change done! But just one question. How did you know it wasn't being effective?

Reply:
I git rebased to a recent commit in master and tried to compile the program.